src/controllers/check.your.answers.controller.ts

```diff
@@ -48,25 +46,30 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
 
 export const post = async (req: Request, res: Response, next: NextFunction) => {
   try {
-    const session = req.session as Session;
     logger.debugRequest(req, `POST ${config.CHECK_YOUR_ANSWERS_PAGE}`);
 
+    const session = req.session as Session;
+    const appData: ApplicationData = getApplicationData(session);
+
     if (isActiveFeature(config.FEATURE_FLAG_ENABLE_REFRESH_TOKEN_29092022)) {
       const accessToken = await refreshToken(req, session);
       logger.infoRequest(req, `New access token: ${accessToken}`);
     }
 
-    const transaction: Transaction = await postTransaction(req, session);
-    logger.infoRequest(req, `Transaction created, ID: ${transaction.id}`);
+    const isSaveAndResumeFeatureActive = isActiveFeature(config.FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022);
+    const transactionID = (!isSaveAndResumeFeatureActive)
```

Comment:
Here and below, I think it would be easier to read code if you removed the ! in front of the `isSaveAndResumeFeatureActive` variable and swapped around the following logic.

Reply:
ok, sure